docs/cloud/dev/spryker-cloud-commerce-os/best-practices/best-practises-jenkins-stability.md

```diff
@@ -0,0 +1,26 @@
+---
+ title: "Best practises: Jenkins stability"
+ description: Improve the stability of the scheduler component.
+ template: best-practices-guide-template
+ last_updated: Jun 10, 2023
+---
+
+Jenkins fulfills the role of a scheduler in the Spryker Application. It is used to run console commands and jobs on a regular basis. In the PaaS setup, the Jenkins instance currently runs the commands configured directly in its container. This means that when you run a command like the following one, it's executed within the Jenkins container, utilizing its resources:
```

Comment:
```suggestion
Jenkins fulfills the role of a scheduler in the Spryker applications. It is used to run repetitive console commands and jobs. In Spryker Cloud Commerce OS, the Jenkins instance runs the commands configured directly in its container. This means that, when you run a command like the following one, it's executed within the Jenkins container, utilizing its resources:
```

Reply:
https://docs.spryker.com/docs/scos/user/intro-to-spryker/contribute-to-the-documentation/spryker-documentation-style-guide/word-list.html